packages/gatsby-theme-apollo-core/src/styles.less

```diff
@@ -12,7 +12,7 @@ body {
 
 @default-margin: 1.45rem;
 
-h1, h2, h3, h4, h5, h6, p, ul, ol, img.screenshot {
+h1, h2, h3, h4, h5, h6, p, ul, ol, img.screenshot, div.mermaid {
```

Comment:
I suppose we could use just the class (`.mermaid`)? Are Mermaid diagrams always `display: block`? Do they always use a `<div>`?